init_root_pos: [0, 0, 0.8]

init_foot_pos_world:
- [0, 0.1, 0]
- [0, -0.1, 0]
init_dof_pos:
  - [0, 0, -0.6, 1.0, -0.4, 0]
  - [0, 0, -0.6, 1.0, -0.4, 0]
  - [0, 0, 0]
  - [0.7, 0.2, -0.1, -1.5 ,0]
  - [0.7, -0.2, 0.1, -1.5 , 0]
